ppr-api/src/ppr_api/resources/financing_statements.py

```diff
@@ -261,17 +292,30 @@ def post(registration_num):
             if not statement.validate_base_debtor(request_json['baseDebtor'], is_staff(jwt)):
                 return resource_utils.base_debtor_invalid_response()
 
-            # TODO: charge a fee.
-
-            # Try to save the change statement: failure throws a business exception.
-            statement = Registration.create_from_json(request_json,
-                                                      model_utils.REG_CLASS_CHANGE,
-                                                      statement,
-                                                      registration_num,
-                                                      account_id)
-            statement.save()
-
-            return statement.json, HTTPStatus.OK  # CREATED
+            # Set up registration and pay.
+            registration = setup_registration_pay(request_json,
+                                                  model_utils.REG_CLASS_CHANGE,
+                                                  statement,
+                                                  registration_num,
+                                                  account_id)
+            invoice_id = str(registration.pay_invoice_id)
+
+            # Try to save the change statement: failure throws an exception.
```

Comment:
Can/should the registration save be abstracted to a function?

Reply:
Great suggestion, I have made this change in a second commit.